MediaGalleryUi/view/adminhtml/web/js/image/image-edit.js

```diff
@@ -49,6 +82,7 @@ define([
                     this.images[imageId] = imageDetails[imageId];
                     this.image(this.images[imageId]);
                     this.openEditImageDetailsModal();
+                    this.getKeywordsOp();
```

Comment:
To avoid this call, simply make sure the keywords component is rendering options based on the imported image